templates/app.html

```diff
@@ -40,7 +40,6 @@
   <!-- Styles -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="static/css/dist/app.min.css" rel="stylesheet">
 
   <!-- <link rel="stylesheet" href="{{ url_for('static', filename='css/dist/bundle.css') }}"> -->
   <link rel="stylesheet" href="http://127.0.0.1:8080/static/css/dist/bundle.css">
```

Comment:
Also, will `bundle.css` work remotely? The href seems to be hardcoded to `http://127.0.0.1:8080/static/css/dist/bundle.css` which makes it work when running webpack locally.